labman/gui/handlers/process_handlers/test/test_sample_plating_process.py

```diff
@@ -100,12 +108,77 @@ def test_sample_plating_process_handler_patch_request(self):
         # Test commenting a well
         self.assertIsNone(tester.notes)
         obs = sample_plating_process_handler_patch_request(
-            user, 10, 'replace', '/well/8/1/notes', 'New Notes', None)
+            self.user, 10, 'replace', '/well/8/1/1/notes', 'New Notes', None)
         self.assertEqual(tester.notes, 'New Notes')
         obs = sample_plating_process_handler_patch_request(
-            user, 10, 'replace', '/well/8/1/notes', '  ', None)
+            self.user, 10, 'replace', '/well/8/1/1/notes', '  ', None)
         self.assertIsNone(tester.notes)
 
+    def test_patch_with_specimen_id(self):
+        # Test success
```

Comment:
I'm confused by this comment ... the tests right below this comment are just showing the well is what we expect it to be when we start the test, right?  Maybe this comment could be expanded a little bit to make it more obvious?